product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx

```diff
@@ -1,28 +1,28 @@
 ---
 title: Optimizing subscriber-only groups
 navTitle: Optimizing
-description: 'Optimizing subscriber-only groups in EDB Postgres Distributed'
+description: 'Optimizing subscriber-only groups in EDB Postgres Distributed.'
 ---
 
 With PGD 5.6 and later, it's possible to optimize the topology of [subscriber-only groups](overview). 
 
-In this optimized topology, a small number of fully active nodes, the write leaders of the data groups, replicate changes to the group leaders of subscriber-only groups, these group leaders then replicate changes to the other members of its subscriber-only group.
+In this optimized topology, a small number of fully active nodes&mdash;the write leaders of the data groups&mdash;replicate changes to the group leaders of subscriber-only groups. These group leaders then replicate changes to the other members of its subscriber-only group.
```

Comment:
Is this what was meant? I didn't think the commas were enough to set off the phrase if so.

Reply:
sure.